src/control/security/auth_sys.go

```diff
@@ -52,42 +63,46 @@ func HashFromToken(token *auth.Token) ([]byte, error) {
 	return hashBytes, nil
 }
 
+func sysNameToPrincipalName(name string) string {
+	return name + "@"
+}
+
 // AuthSysRequestFromCreds takes the domain info credentials gathered
 // during the gRPC handshake and creates an AuthSys security request to obtain
 // a handle from the management service.
-func AuthSysRequestFromCreds(creds *DomainInfo) (*auth.Credential, error) {
+func AuthSysRequestFromCreds(ext UserExt, creds *DomainInfo) (*auth.Credential, error) {
 	if creds == nil {
 		return nil, errors.New("No credentials supplied")
 	}
 
-	uid := strconv.FormatUint(uint64(creds.creds.Uid), 10)
-	userInfo, _ := user.LookupId(uid)
-	groups, _ := userInfo.GroupIds()
+	userInfo, _ := ext.LookupUserID(creds.creds.Uid)
```

Comment:
why ignore error here?

Reply:
Just copied the old logic - @dpquigl wrote the initial implementation, so maybe he can comment on why.

Reply:
When I originally looked up examples of how todo this it ignored the error code as well. I'm not sure if it can actually fail on Unix I think the error is for mostly returning failures on windows

Reply:
is that something we want to leave to chance? I'm fine if you have a high confidence level in it not failing on Windows but perhaps we want to note that assumption somewhere?

Reply:
We can probably check it just to make sure. No harm in doing so.

Reply:
Happy to address that in a follow-on PR. Added a ticket so we don't forget: https://jira.hpdd.intel.com/browse/DAOS-2542